packages/destination-actions/src/destinations/clevertap/index.ts

```diff
@@ -10,30 +10,28 @@ const destination: DestinationDefinition<Settings> = {
   name: 'Clevertap (Actions)',
   slug: 'actions-clevertap',
   mode: 'cloud',
-  description: '<p data-renderer-start-pos="72">CleverTap is a customer engagement and retention platform that provides the functionality to integrate app analytics and marketing. The platform helps customers increase user engagement in three ways:</p>' +
-    '<ul class="ak-ul" data-indent-level="1">' +
-    '<li><p data-renderer-start-pos="276">Tracks actions users are taking and analyzes how people use the product.</p></li>' +
-    '<li><p data-renderer-start-pos="352"><a class="sc-1ko78hw-0 fiVZLH" href="https://docs.clevertap.com/docs/segments" title="https://docs.clevertap.com/docs/segments" data-renderer-mark="true">Segment</a> users based on their actions and run targeted <a class="sc-1ko78hw-0 fiVZLH" href="https://docs.clevertap.com/docs/intro-to-campaigns" title="https://docs.clevertap.com/docs/intro-to-campaigns" data-renderer-mark="true">campaigns</a> to these segments.</p></li>' +
-    '<li><p data-renderer-start-pos="438"><a class="sc-1ko78hw-0 fiVZLH" href="https://docs.clevertap.com/docs/intro-to-reports" title="https://docs.clevertap.com/docs/intro-to-reports" data-renderer-mark="true">Analyze</a> each campaign to understand its effect on user engagement and business metrics.</p></li>' +
-    '</ul>',
+  description: 'CleverTap is a customer engagement and retention platform that provides the functionality to integrate app analytics and marketing. The platform helps customers increase user engagement in three ways:\n' +
```

Comment:
I believe Markdown uses `<br>` html for line breaks.. Don't think `\n` will work here.

```suggestion
  description: 'CleverTap is a customer engagement and retention platform that provides the functionality to integrate app analytics and marketing. The platform helps customers increase user engagement in three ways: <br>' +
```

Reply:
pushed these changes please review again.